dask/array/core.py

```diff
@@ -2852,10 +2838,25 @@ def choose(a, choices):
 """.strip()
 
 
+@wraps(np.argwhere)
+def argwhere(a):
+    from .creation import indices
+
+    nz = (a != 0).flatten()
```

Comment:
Technically this doesn't support strings. Not sure of an easy fix. Partially because NumPy treats things like `" "` as `False` when Python does not. Also casting a string array to `bool`s doesn't work either. So not entirely sure how to address it. Maybe it is best left alone until someone has a use case and decides to fix this.